### ISSUE TITLE
Can no longer upgrade

### ISSUE THREAD

**Issue by kswope** [phusion/passenger-docker#139]
The current version is getting pretty far behind now so I'm trying to run this in my dockerfile, as shown in the README

```
RUN apt-get update && apt-get upgrade -y -o Dpkg::Options::="--force-confold"
```

This now gets stuck, and I'm not sure where, all I can tell you is that this is what is at the end when I tail the contain log as it builds.  And there's an out of control java process running.

```
Setting up dpkg-dev (1.17.5ubuntu5.7) ...
Setting up nginx-extras (1:1.10.1-8.5.0.29~trusty1) ...
invoke-rc.d: policy-rc.d denied execution of start.
Processing triggers for libc-bin (2.19-0ubuntu6.9) ...
Processing triggers for initramfs-tools (0.103ubuntu4.3) ...
Processing triggers for ca-certificates (20160104ubuntu0.14.04.1) ...
Updating certificates in /etc/ssl/certs... 19 added, 19 removed; done.
Running hooks in /etc/ca-certificates/update.d....
Adding debian:CA_WoSign_ECC_Root.pem
Adding debian:CFCA_EV_ROOT.pem
Adding debian:COMODO_RSA_Certification_Authority.pem
Adding debian:Certification_Authority_of_WoSign_G2.pem
Adding debian:Certinomis_-_Root_CA.pem
Adding debian:Entrust_Root_Certification_Authority_-_EC1.pem
Adding debian:Entrust_Root_Certification_Authority_-_G2.pem
Adding debian:GlobalSign_ECC_Root_CA_-_R4.pem
Adding debian:GlobalSign_ECC_Root_CA_-_R5.pem
Adding debian:IdenTrust_Commercial_Root_CA_1.pem
Adding debian:IdenTrust_Public_Sector_Root_CA_1.pem
Adding debian:OISTE_WISeKey_Global_Root_GB_CA.pem
Adding debian:S-TRUST_Universal_Root_CA.pem
Adding debian:Staat_der_Nederlanden_EV_Root_CA.pem
Adding debian:Staat_der_Nederlanden_Root_CA_-_G3.pem
Adding debian:TÜRKTRUST_Elektronik_Sertifika_Hizmet_Sağlayıcısı_H5.pem
Adding debian:TÜRKTRUST_Elektronik_Sertifika_Hizmet_Sağlayıcısı_H6.pem
Adding debian:USERTrust_ECC_Certification_Authority.pem
Adding debian:USERTrust_RSA_Certification_Authority.pem

```

**Comment by kswope** [comment]
This might have something to do with an older version of docker

currently running
```
Docker version 1.9.1, build a34a1d5
```

I'll report back if an upgrade helps

**Comment by kswope** [comment]
Upgrading worked

```
Docker version 1.11.2, build b9f10c9
```

**Comment by FooBarWidget** [comment]
This has been fixed in 0.9.19.
